docs/test-drive.md

```diff
@@ -1,6 +1,6 @@
 # Take a test drive
 
-You can easily try out Netlify CMS by running it on a pre-configured starter site. Our example in the intro is the [Kaldi small business Hugo template](https://github.com/netlify-templates/kaldi-hugo-cms-template). Use the deploy button below to build and deploy your own copy of the repository:
+You can easily try out Netlify CMS by running it on a pre-configured starter site. Our example in the [intro](https://github.com/netlify/netlify-cms/blob/master/docs/intro.md) is the [Kaldi small business Hugo template](https://github.com/netlify-templates/kaldi-hugo-cms-template). Use the deploy button below to build and deploy your own copy of the repository:
```

Comment:
Yeah, they should all be relative URLs if they're pointing to other files in the `/docs` directory. If they're outside that, though, like `CONTRIBUTING.md`, they should keep the GitHub URL.